EDDiscovery/EDDiscovery.csproj

```diff
@@ -535,11 +534,11 @@
     <Compile Include="UserControls\UserControlScan.Designer.cs">
       <DependentUpon>UserControlScan.cs</DependentUpon>
     </Compile>
-    <Compile Include="UserControls\UserControlSpanel.cs">
+    <Compile Include="UserControls\UserControlSPanel.cs">
```

Comment:
This rename (without the corresponding file rename) is causing a travis build failure